src/tlo/methods/tb.py

```diff
@@ -1933,6 +1933,74 @@ def apply(self, person_id, squeeze_factor):
             return ACTUAL_APPT_FOOTPRINT
 
 
+class HSI_Tb_ClinicalDiagnosis(HSI_Event, IndividualScopeEventMixin):
+    """
+    This is a clinical diagnosis appt which is called when other tests have not been
+    available and only a clinical diagnosis is required
+
+    * it does not include any of the routine tests for TB or HIV
+    therefore property tb_date_tested is not updated
+    * It only requires 0.5 footprint of Under5OPD since it will almost exclusively
+    be used for children unable to get xrays following initial diagnostic consultations
+    """
+
+    def __init__(self, module, person_id, suppress_footprint=False):
+        super().__init__(module, person_id=person_id)
+        assert isinstance(module, Tb)
+
+        assert isinstance(suppress_footprint, bool)
+        self.suppress_footprint = suppress_footprint
+
+        self.TREATMENT_ID = "Tb_Test_Clinical"
+        self.EXPECTED_APPT_FOOTPRINT = self.make_appt_footprint({"Under5OPD": 0.5})
+        self.ACCEPTED_FACILITY_LEVEL = '1a'
+
+    def apply(self, person_id, squeeze_factor):
+        """ Do the screening and referring process """
+
+        df = self.sim.population.props
+        now = self.sim.date
+        person = df.loc[person_id]
+        test_result = None
+
+        # If the person is dead or already diagnosed, do nothing do not occupy any resources
+        if not person["is_alive"] or person["tb_diagnosed"]:
+            return self.sim.modules["HealthSystem"].get_blank_appt_footprint()
+
+        logger.debug(
+            key="message", data=f"HSI_Tb_ClinicalDiagnosis: person {person_id}"
+        )
+
+        # check if patient has: cough, fever, night sweat, weight loss
+        # if none of the above conditions are present, no further action
+        persons_symptoms = self.sim.modules["SymptomManager"].has_what(person_id)
+        if not any(x in self.module.symptom_list for x in persons_symptoms):
+            return self.make_appt_footprint({})
+
+        # check for presence of all symptoms (clinical diagnosis)
+        if all(x in self.module.symptom_list for x in persons_symptoms):
+            test_result = self.sim.modules["HealthSystem"].dx_manager.run_dx_test(
+                dx_tests_to_run="tb_clinical", hsi_event=self
+            )
```

Comment:
do you want to do anything if a person has some but not all of the symptoms?

Reply:
No, I think the clinical criteria require all 4 symptoms to be present for a clinical diagnosis

Reply:
OK!